src/nvim/drawline.c

```diff
@@ -517,8 +517,8 @@ static void get_sign_display_info(bool nrcol, win_T *wp, winlinevars_T *wlv, int
 static int get_sign_attrs(buf_T *buf, winlinevars_T *wlv, int *sign_num_attrp, int *sign_cul_attrp)
 {
   HlPriAttr line_attrs = { wlv->line_attr,  0 };
-  HlPriAttr num_attrs  = { *sign_num_attrp, 0 };
```

Comment:
These are both `0` when `get_sign_attrs()` is called. Dereferencing them here to initialize another variable is only confusing.